index.md

```diff
@@ -12,35 +12,101 @@ First we set up & capture the content, then we render it in the scaffolding belo
+
+### Team-friendly
+Cockpit is friendly enough for those new to Linux and useful for seasoned admins too.
+
+### Packages included
+[Installing and running Cockpit]({{ site.baseurl }}/running.html) is simple. It's already included in most of the major distributions.
 
 ---
 
-[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png){:.screenshot.zoom}
-### No interference
-Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
+## Integrated
+
+### Plays well with others
+Jumping between the terminal and the web tool is no problem.
+
+Keep using the command line, Ansible, and your other favorite tools and add Cockpit to the mix with no issues.
+
+A service started via Cockpit can be stopped in a terminal. Likewise, if an error occurs in a terminal, it's also in Cockpit's journal.
+
+[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png)
+{:.screenshot.zoom}
+
+### Sign in like normal
+By default, Cockpit uses [your system's normal user logins and privileges]({{ site.baseurl }}/guide/latest/privileges). You don't need to set up any special accounts. Network-wide logins are also supported through [single-sign-on]({{ site.baseurl }}/guide/latest/sso) and other [authentication]({{ site.baseurl }}/guide/latest/authentication) techniques.
+
+### Self-contained
+You don't have to worry about setting up a webserver just to use Cockpit.
+
+### Uses existing APIs
+Cockpit uses APIs that already exist on the system. It doesn't reinvent subsystems or add a layer of its own tooling.
+
+### Efficient
+
+Cockpit only uses memory and CPU when active. When inactive, there is no extra load on your server.
 
 ---
 
-[![Dashboard screenshot]({{ site.baseurl }}/images/site/screenshot-dashboard.png)]({{ site.baseurl }}/images/site/screenshot-dashboard.png){:.screenshot.zoom}
+## Glanceable
+
+### System overview
+Immediately understand the health of your server. Cockpit's overview page shows current statistics and the status of your system.
+
 ### Multi-server
-You can monitor and administer several servers at the same time. Just add it easily and your server will look after its buddies.
+Monitor and administer [several servers]({{ site.baseurl }}/guide/latest/feature-machines.html) at the same time. Add new hosts and your main server will look after its buddies.
+
+[![Dashboard screenshot]({{ site.baseurl }}/images/site/screenshot-dashboard.png)]({{ site.baseurl }}/images/site/screenshot-dashboard.png)
+{:.screenshot.zoom}
+
+### Troubleshoot
+Fix pesky problems with ease.
+
+- Inspect the state of virtual machines
```

Comment:
To avoid saying "Inspect" twice, how about "Spot and react to misbehaving virtual machines"

Reply:
I'm all for reducing the repeating word redundancy. I'll update.

I also swapped the order so network issues is first. Two reasons: Network issues are more universal and the points go from shorter to longer now, so it's more visually pleasing.